### PR TITLE
Fix/patient issues api

### PR DESCRIPTION
**Pull request by johnny2smooth**
Fixed the main issue of issue reports not repopulating back into the issues tab after reviewed.

I still have a few questions (mainly about the use of the patients array provided by context and how that is updated upon markAsReviewed), but we can chat about those another time and maybe create another branch for those issues.

### CODE REVIEW

File: src/Practitioner/ReviewPatients/HomeTopBar.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { Search, Add } from '@material-ui/icons';
+import Colors from '../../Basics/Colors';
+import { Link } from 'react-router-dom';
+
+export default function TopBar() {
+  const { t } = useTranslation('translation');
+  return (
+    <Box bgcolor="white" borderBottom="solid 1px lightgray" padding=".5em 1em">
+      <Grid alignItems="center" container>
+        {/* Why does it say all patients? */}
```
Comment (kylegoodwin) [inline src/Practitioner/ReviewPatients/HomeTopBar.js:14]: This is prolly cause its an existing translation we are using before it was used in this context, good call tho

File: src/Practitioner/ReviewPatients/checkWasToday.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { DateTime } from 'luxon';
+
```
Comment (kylegoodwin) [inline src/Practitioner/ReviewPatients/checkWasToday.js:2]: I would move this function here

src/Utility/TimeUtils.js

Trying to collect common util code there

**Review by kylegoodwin — CHANGES_REQUESTED** [review]
This looks good, we can maybe walk through some testing together on monday

File: src/Practitioner/ReviewPatients/checkWasToday.js (hunk 2 above)
Comment (kylegoodwin) [reply src/Practitioner/ReviewPatients/checkWasToday.js:2]: Oh sorry this showed up in the wrong spot. I meant this whole file

File: src/Practitioner/ReviewPatients/HomeTopBar.js (hunk 1 above)
Comment (johnny2smooth) [reply src/Practitioner/ReviewPatients/HomeTopBar.js]: Changed to 'Daily Reports'

File: src/Practitioner/ReviewPatients/checkWasToday.js (hunk 2 above)
Comment (johnny2smooth) [reply src/Practitioner/ReviewPatients/checkWasToday.js]: Moved, now imported from TimeUtils, and deleted checkWasToday

**Pull request merged**